Automattic/mShots
# Update tweet screenshot mechanism

This PR updates the tweet screenshot mechanism to take a whole page screenshot (not exactly, read below) instead of a specific element, whose selector is prone to change.

This won't be a "whole page screenshot" as in it takes a screenshot of the `body` element instead, since taking a screenshot using `page.screenshot({...})` causes a page reload in this case, for some reason.

**Testing instructions**

- Try taking a screenshot of a tweet, e.g. http://localhost:8888/mshots/v1/https%3A%2F%2Fplatform.twitter.com%2Fembed%2Findex.html%3Fdnt%3Dfalse%26embedId%3Dtwitter-widget-0%26frame%3Dfalse%26hideCard%3Dfalse%26hideThread%3Dtrue%26id%3D1344758902211055616%26lang%3Den
- Verify the screenshot is taken correctly, meaning all images are loaded and that the blank background has been cropped out
